tools/src/commands/gen_config/mod.rs

```diff
@@ -181,14 +190,30 @@ pub fn main(opts: Opts) -> Result<()> {
         println!("Creating a new artifact");
         Default::default()
     });
+    if let SxgCertConfig::CreateAcmeAccount(acme_config) = &input.certificates {
+        if artifact.acme_account.is_none() {
+            let (acme_private_key, acme_account) = tokio_block_on(create_acme_key_and_account(
+                acme_config,
+                &input.sxg_worker.html_host,
+            ))?;
+            artifact.acme_account = Some(acme_account);
+            artifact.acme_private_key = Some(acme_private_key);
```

Comment:
Fix this.

Reply:
Fixed by [the latest commit](https://github.com/google/sxg-rs/pull/323/commits/a93ac26289562bc9a11f6a2335da9959ef969699).